packages/sdk/src/across/clients/bridgePool.ts

```diff
@@ -253,6 +256,11 @@ function joinPoolState(
     SECONDS_PER_YEAR
   );
   const estimatedApr = calcApr(exchangeRatePrevious, exchangeRateCurrent, secondsElapsed, SECONDS_PER_YEAR);
+  let projectedApr = "N/A";
+  const rateModel = getRateModel(poolState.l1Token);
+  if (rateModel) {
+    projectedApr = fromWei(calculateInstantaneousRate(rateModel, poolState.liquidityUtilizationCurrent));
```

Comment:
Do we need to multiply by the utilization?

Reply:
yea, good call

Reply:
do we also have to mutliply the current apr/apy value by utilization?

Reply:
this makes projected apr much lower fyi 
![image](https://user-images.githubusercontent.com/4429761/144039302-f9faa203-9b0f-4914-a57f-7cfc5d3c7056.png)

Reply:
@daywiss 

> this makes projected apr much lower fyi

I think that's expected. The pool utilization for ETH is 20% right now, which is pretty low, meaning we'd be charging a low fee and it's being charged on a very small amount of the capital in the pool (since very little is loaned out).

We should probably chat with other folks to ensure that we want to show this.

> do we also have to mutliply the current apr/apy value by utilization?

I don't think so. For the current, we're measuring the contract's fee appreciation divided across all shares, so this is already handled by the contract's internal math.